src/i18n/locales/es.json

```diff
@@ -203,7 +206,7 @@
         "balance": "<span>Dispones de:</span><span2> {{ balance }} tokens</span2>",
         "census": "Censo",
         "census_preview_is_shortened": "(la lista previa está limitada a {{ limit }} resultados)",
-        "census_total": "",
+        "census_total": "Eliminar direccion con indice {{ index }}",
         "census_total_one": "<span>Votantes:</span><span>{{ count }} votante</span>",
         "census_total_many": "",
         "census_total_other": "<span>Votantes:</span><span>{{ count }} votantes</span>",
```

Comment:
@gerouvi careful! You re-used an existing plural translation. Please define a new key for `census_total`. Also, "índice" has a missing accent.